monkey/tests/conftest.py

```diff
@@ -11,3 +11,13 @@
 @pytest.fixture(scope="session")
 def data_for_tests_dir(pytestconfig):
     return Path(os.path.join(pytestconfig.rootdir, "monkey", "tests", "data_for_tests"))
+
+
+@pytest.fixture(scope="session")
+def stable_file(data_for_tests_dir) -> Path:
```

Comment:
We should add a typehint that `data_for_tests_dir` is a `Path` as well. If we refactor it to not be a path, the syntax of `some_path / "some_file.txt"` will break

Reply:
`data_for_tests_dir` isn't actually a Path in this case, it's a pytest fixture. There are type hints for fixtures, but you have to import them from a file that's intended to be private. https://github.com/pytest-dev/pytest/issues/5981#issuecomment-790261424